core/src/components/textarea/textarea.tsx

```diff
@@ -252,6 +257,18 @@ export class Textarea implements ComponentInterface {
     });
   }
 
+  /**
+   * Emits an `ionChange` event.
+   *
+   * This API should be called for user committed changes.
+   * This API should not be used for external value changes.
+   */
+  private emitValueChange() {
+    const { value } = this;
+    const newValue = value == null ? value : value.toString();
```

Comment:
Can we either make this an explicit null + undefined check or add a comment that notes the `==` checks both null and undefined?

Reply:
This change was made to align with [`ion-input`](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/input/input.tsx#L257). Do we have a preference on the approach - I can update both input and textarea to match.

Personally, I feel that using the loose equality operator for checking for `null`/`undefined` is a fairly common practice; so I would opt for the comment if we feel it needs calling out.

Reply:
Yeah a comment is probably fine. 👍